plaidml/bridge/keras/__init__.py

```diff
@@ -679,7 +679,7 @@ def conv3d_transpose(x,
 @_log_call
 def count_params(x):
     result = 1
-    for dim in x.tensor.shape.int_dims:
+    for dim in x.tensor.compute_shape().into_TensorShape().sizes:
```

Comment:
I don't think we need to convert to `TensorShape` here, but `compute_shape()` is probably necessary.

Reply:
Is it possible to use `x.bind_dims()` here instead and compute the product symbolically I wonder?

Reply:
Keras expects `count_params` to return something that's more-or-less a python integer (specifically, going by the uses in the Keras source code, if `n` is a python integer, then `n == X.count_params()` needs to behave as expected and `int(X.count_params())` needs to behave as expected). You could do something like
```
shape = edsl.TensorDims(I.rank)
I.bind_dims(*shape)
ret = 1
for dim in shape:
    ret *= dim
return ret
```
but will that cast and compare to python ints correctly?

Reply:
OK, no, we have to call` compute_shape()` then.